packages/app/src/app/pages/Dashboard/Content/CreateNewSandbox/Modal/index.js

```diff
@@ -55,123 +54,73 @@ export default class Modal extends React.PureComponent {
-              {ts.map(t => (
-                <Template
-                  width={Math.floor(paddedWidth / mainTemplatesPerRow - 16)}
-                  key={t.name}
-                  template={t}
-                  selectTemplate={this.selectTemplate}
-                />
-              ))}
-            </Templates>
-          ))}
+export default class Modal extends React.PureComponent {
+  state = {
+    selectedTab: 0,
+  };
 
-          {rows.map((ts, i) => (
-            // eslint-disable-next-line react/no-array-index-key
-            <Templates style={{ fontSize: '.8rem' }} key={i}>
-              {ts.map(t => (
-                <Template
-                  small
-                  width={Math.floor(paddedWidth / templatesPerRow - 16)}
-                  key={t.name}
-                  template={t}
-                  selectTemplate={this.selectTemplate}
-                />
-              ))}
-            </Templates>
-          ))}
+  setTab = tab => this.setState({ selectedTab: tab });
 
-          <Title>Server Templates</Title>
-          {mainServerRows.map((ts, i) => (
-            // eslint-disable-next-line react/no-array-index-key
-            <Templates key={i}>
-              {ts.map(t => (
-                <Template
-                  width={Math.floor(paddedWidth / mainTemplatesPerRow - 16)}
-                  key={t.name}
-                  template={t}
-                  selectTemplate={this.selectTemplate}
-                />
-              ))}
-            </Templates>
-          ))}
+  selectTemplate = template => {
+    this.props.createSandbox(template);
+  };
 
-          {serverRows.map((ts, i) => (
-            // eslint-disable-next-line react/no-array-index-key
-            <Templates style={{ fontSize: '.8rem' }} key={i}>
-              {ts.map(t => (
-                <Template
-                  small
-                  width={Math.floor(paddedWidth / templatesPerRow - 16)}
-                  key={t.name}
-                  template={t}
-                  selectTemplate={this.selectTemplate}
-                />
-              ))}
-            </Templates>
-          ))}
+  render() {
+    const { forking = false, closing = false } = this.props;
 
-          <Title>Presets</Title>
-          {typescriptRows.map((ts, i) => (
-            // eslint-disable-next-line react/no-array-index-key
-            <Templates style={{ fontSize: '.8rem' }} key={i}>
-              {ts.map(t => (
-                <Template
-                  small
-                  width={Math.floor(paddedWidth / templatesPerRow - 16)}
-                  key={t.name}
-                  template={t}
-                  selectTemplate={this.selectTemplate}
-                  subtitle={`Using ${t.variantName} template`}
-                />
-              ))}
-            </Templates>
+    const { selectedTab } = this.state;
+    return (
+      <Fragment>
+        <TabContainer>
+          {availableTemplates.map((t, i) => (
```

Comment:
you could deconstruct `name` inline as you don't use the entire keys

```js
{availableTemplates.map(({ name }, i) => (
```

Reply:
Same :)